source/PlayerInfo.cpp

```diff
@@ -2634,6 +2634,10 @@ void PlayerInfo::RegisterDerivedConditions()
 	flagshipModelProvider.SetHasFunction(flagshipModelFun);
 	flagshipModelProvider.SetGetFunction(flagshipModelFun);
 
+	auto &&nameProvider = conditions.GetProviderNamed("playername: " + firstName + " " + lastName);
```

Comment:
Are you sure this works? Wouldn't this create a condition literally named `playername: first last`? You want to use a prefixed provider like above I believe.

Reply:
Well it would probably work, but i can do a prefix if you want me to.
I will see when I get to this got like six other "larger" code PR's ongoing

Reply:
Added